app/controllers/api/v1/recordings_controller.rb

```diff
@@ -37,6 +38,19 @@ def destroy
         render_json(status: :ok)
       end
 
+      # PUT/PATCH /api/v1/recordings/:recording_id.json
+      # Returns: { data: Array[serializable objects(recordings)] , errors: Array[String] }
+      # Does: Update the recording name.
+      def update
+        new_name = recording_params[:name]
+        return render_json errors: [Rails.configuration.custom_error_msgs[:missing_params]], status: :bad_request if new_name.blank?
+
+        BigBlueButtonApi.new.update_recordings record_id: @recording.record_id, meta_hash: { 'meta_recording-name': new_name }
```

Comment:
I think to avoid confusion, we should call this `meta_name` - the same thing as v2, so that when we import the recordings from old rooms, their names don't change